### ISSUE TITLE
Conversion of ONNX model error

### ISSUE THREAD

**Issue by gt1485a** [luxonis/blobconverter#33]
I am converting an ONNX model and I get the following error:

`python3 -m blobconverter --onnx-model /home/dso-dat-dev6/dso/mde.onnx --shaves 6
Downloading /home/dso-dat-dev6/.cache/blobconverter/mde_openvino_2021.4_6shave.blob...
{
    "exit_code": 1,
    "message": "Command failed with exit code 1, command: /opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/myriad_compile -m /tmp/blobconverter/c7d2ecf91d79439fbc8597d42516336d/mde/FP16/mde.xml -o /tmp/blobconverter/c7d2ecf91d79439fbc8597d42516336d/mde/FP16/mde.blob -c /tmp/blobconverter/c7d2ecf91d79439fbc8597d42516336d/myriad_compile_config.txt -ip U8",
    "stderr": "Unsupported pad value\n",
    "stdout": "Inference Engine: \n\tIE version ......... 2021.4.0\n\tBuild ........... 2021.4.0-3839-cd81789d294-releases/2021/4\n\u001b[1;33m[Warning][VPU][Config] Deprecated option was used : VPU_MYRIAD_PLATFORM\u001b[0m\n"
}`

How should i go about debugging this issue?

Thanks

**Comment by tersekmatija** [comment]
Adding unsupported layers might not be necessary. But you will have to investigate which layer is causing this. Simply replacing the pad value or the problematic layer with supported layer might be enough.

**Comment by gt1485a** [comment]
So running locally i get a different error which is strange. The error locally is:
`workbench@71b1de6d19da:/$ /opt/intel/openvino_2021.4.689/deployment_tools/tools/compile_tool/compile_tool -m /home/workbench/.workbench/models/14/original/mde.onnx -ip U8 -d MYRIAD -VPU_NUMBER_OF_SHAVES 4 -VPU_NUMBER_OF_CMX_SLICES 4
Inference Engine: 
	IE version ......... 2021.4.1
	Build ........... 2021.4.1-3926-14e67d86634-releases/2021/4

[WARN] 2022-03-17T17:56:35z frontend/onnx_import/src/op/constant.cpp 35	
Could not create an nGraph Constant for an ONNX Constant node. Constant with a 0 value was created instead.
Verify if the ONNX Constant node contains a correct number of elements matching the node's shape. 
Detailed error:
Check 'values.size() == 1 || values.size() == shape_size(m_shape)' failed at core/include/ngraph/op/constant.hpp:59:
While validating node 'v0::Constant Constant_504 () -> (f32{})' with friendly_name 'Constant_504':
Did not get the expected number of literals for a constant of shape Shape{} (got 0, expected 1).

[WARN] 2022-03-17T17:56:35z frontend/onnx_import/src/op/constant.cpp 35	
Could not create an nGraph Constant for an ONNX Constant node. Constant with a 0 value was created instead.
Verify if the ONNX Constant node contains a correct number of elements matching the node's shape. 
Detailed error:
Check 'values.size() == 1 || values.size() == shape_size(m_shape)' failed at core/include/ngraph/op/constant.hpp:59:
While validating node 'v0::Constant Constant_506 () -> (f32{})' with friendly_name 'Constant_506':
Did not get the expected number of literals for a constant of shape Shape{} (got 0, expected 1).

Check '(node.get_outputs_size() == 1)' failed at frontend/onnx_import/src/op/batch_norm.cpp:78:
While validating ONNX node '<Node(BatchNormalization): BatchNormalization_28>':
Training mode of BatchNormalization is not supported.`

When i convert the PyTorch to ONNX I get this error about BatchNormalization which seems shouldn't be in the output model anyway. Will open an issue over on isl-org/dpt unless you have run into this before...

`/Users/haldiry/Documents/VLS-Gitlab/ai-platform-monocular-depth-estimation/mono_depth/lib/python3.9/site-packages/torch/onnx/symbolic_helper.py:770: UserWarning: ONNX export mode is set to inference mode, but operator batch_norm is set to training  mode. The operators will be exported in training , as specified by the functional operator.
  warnings.warn("ONNX export mode is set to " + training_mode +`

**Comment by VanDavv** [comment]
@gt1485a you're running a different command than blobconverter, hence different output. Blobconverter first converts the model to IR format and then performs compilation to MyriadX blob.
I don't know if the `compile_tool` does the same thing, but since blobconverter failed while running `deployment_tools/inference_engine/lib/intel64/myriad_compile`, it successfully managed to produce IR format for your ONNX model, so maybe you could try doing the same locally

**Comment by Erol444** [comment]
Closing due to inactivity..
